keg_bouncer/model/mixins.py

```diff
@@ -1,21 +1,37 @@
 from __future__ import absolute_import
 
 import sqlalchemy as sa
+from sqlalchemy.inspection import inspect
 from sqlalchemy.ext.declarative import declared_attr
 import sqlalchemy.orm as saorm
 
-import flask_user
+from . import entities as ents
 
-from .entities import (
-    UserGroup,
-    joined_permission_query,
-    make_user_to_user_group_link,
-    user_group_bundle_map,
-    user_group_permission_map,
-)
 
+class KegBouncerMixin(object):
+    _invalid_primary_key_error = AttributeError(
```

Comment:
Why did you create a class variable for this exception? That seems like an odd thing to do.